File: LC80.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        
        n = len(nums)

        if n < 3:
            return n
        
        i = 2; j = 2

        while j < n:
            if nums[j] != nums[i-2]:
                nums[i] = nums[j]
                i += 1
            j += 1
        
        return i

File: LC80.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();

        if(n < 3){
            return n;
        }
        
        int i = 2, j = 2;

        while(j < n){
            if(nums[j] != nums[i-2])
                nums[i++] = nums[j];
            j++;
        }

        return i;
    }
};
